### DISCUSSION TITLE
着色器与字体更新

### DISCUSSION THREAD

**Discussion by nanachisol** [dyphire/mpv-config#133] [General]
NotoSerifCJK
https://github.com/notofonts/noto-cjk/blob/main/Serif/OTC/NotoSerifCJK-Bold.ttc
adaptive-sharpen与SSimDownscaler
https://gist.github.com/igv
nnedi3，ravu 或者可以换用shaders in compute
https://github.com/bjin/mpv-prescalers
nlmeans 或者其他
https://github.com/AN3223/dotfiles/tree/master/.config/mpv/shaders
至于其余着色器，精力所限，实在不能一一查证。

**Comment by dyphire** [comment]
> NotoSerifCJK 
https://github.com/notofonts/noto-cjk/blob/main/Serif/OTC/NotoSerifCJK-Bold.ttc

字体体积太大，正常情况下不会再推送仓库更新。播放器和字幕一般也用不到后续更新的扩展区字形

> adaptive-sharpen 与 SSimDownscaler
https://gist.github.com/igv

查看 Revisions 后 adaptive-sharpen 并没有新的更新，SSimDownscaler 更新很久以前我就自行更改过

> nnedi3，ravu 或者可以换用 shaders in compute 
https://github.com/bjin/mpv-prescalers 

不使用 compute 的理由见相关着色器更新日志 bb49059
> nlmeans 或者其他
https://github.com/AN3223/dotfiles/tree/master/.config/mpv/shaders 

nlmeans 只是实验性着色器，放进来仅供测试。性能太差不适合日常使用，无意更新

> 至于其余着色器，精力所限，实在不能一一查证。

想到再说吧

**Comment by nanachisol** [comment]
adaptive-sharpen 
```
#define curve_height    0.5                  // Main control of sharpening strength [>0]
                                             // 0.3 <-> 2.0 is a reasonable range of values
```
SSimDownscaler 

` return vec4(avg[0], mix(sqrt((Sh + sigma_nsq) / (Sl + sigma_nsq)) * (1. + oversharp), clamp(Sh / Sl, 0., 1.), int(Sl > Sh)));`
以上两行是自行更改过的？

**Reply by dyphire** [reply]
看错了，SSimDownscaler 之前修的另一个问题。之后更下
